src/ui/components/LedgerAdmin.js

```diff
@@ -169,14 +209,33 @@ export const LedgerAdmin = (): ReactNode => {
         >
           save ledger to disk
         </Button>
-        {selectedId && <Button onClick={resetIdentity}>New identity</Button>}
       </ButtonGroup>
+      {selectedId && (
+        <Button onClick={resetIdentity} className={classes.backButton}>
+          Back
+        </Button>
+      )}
       {selectedId && (
         <>
           <AliasView selectedId={selectedId} />
           <IdentityMerger selectedId={selectedId} />
         </>
-      )}
+      )}{" "}
+      <div className={classes.spreadRow}>
+        <h3>
+          Identities{" "}
+          {ledger.accounts().length > 0 && (
```

Comment:
Instead of calling ledger.accounts() again, you can use the memoized `accounts` variable defined in the comment I made above on Line 76

Reply:
Good catch, fixed.